### PR TITLE
Update Chrome driver v115 onwards with new strategy

### PR DESCRIPTION
**Pull request by Carolina0802**
- Modifications are made to the ChromeDriver download logic in versions greater than or equal to 115, the changes have been made in the SEResourcesIntaller and DALUriDB scripts.

### CODE REVIEW

File: src/SEResourcesInstaller.ps1 [hunk 1]
```diff
@@ -34,173 +34,199 @@
...
+    else {
+      $packageName = "$driverOutput$packAlias" + "_pack.zip"
 			(New-Object System.Net.WebClient).DownloadFile("$resourceUrl", "$packageName")
-			Start-Sleep -m 800
-			Expand-Archive -Path "$packageName" -DestinationPath "$driverOutput" -Force	
-		}
+      Start-Sleep -m 800
+
+      function DownloadChromePlatform() {
+
```
Comment (dloperab) [inline src/SEResourcesInstaller.ps1]: Remove enters

File: src/SEResourcesInstaller.ps1 [hunk 2]
```diff
@@ -34,173 +34,199 @@
...
+        
+      
+        if ($platform -eq "WIN") {
+          $systemChr = "chromedriver-win32" 
+        }
+        else {
+          if ($platform -eq "MAC") {
+            $systemChr = "chromedriver-mac-x64" 
+          }
+          else {
```
Comment (dloperab) [inline src/SEResourcesInstaller.ps1]: Could you use an `else if` like this:

```
if ($platform -eq "WIN") {
  $systemChr = "chromedriver-win32" 
} else if ($platform -eq "MAC") {
  $systemChr = "chromedriver-mac-x64" 
} else { 
  $systemChr = "chromedriver-linux64" 
}
```

File: src/SEResourcesInstaller.ps1 [hunk 3]
```diff
@@ -34,173 +34,199 @@
...
+          }
+          else { 
+            if ($platform -eq "LNX") {
+              $systemChr = "chromedriver-linux64" 
+            }
+          }
+        }
+        return $systemChr
+      }
+      function DownloadChrome() {
```
Comment (dloperab) [inline src/SEResourcesInstaller.ps1:130]: Add an enter

File: src/SEResourcesInstaller.ps1 [hunk 4]
```diff
@@ -34,173 +34,199 @@
...
+          Copy-Item "$driverOutput\$systemChr\chromedriver.exe"  $driverOutput 
+          Remove-Item "$driverOutput\$systemChr\" -Force -Recurse
+          Write-Host "Download ChromeDriver version > = 115"
+        }
+        else {
+          Expand-Archive -Path "$packageName" -DestinationPath "$driverOutput" -Force 
+          Write-Host "ChromeDriver version < 115"
+        }           	
+      } 
+      DownloadChrome
```
Comment (dloperab) [inline src/SEResourcesInstaller.ps1:148]: Add an enter

File: src/SEResourcesInstaller.ps1 [hunk 5]
```diff
@@ -34,173 +34,199 @@
...
+          Write-Host "Download ChromeDriver version > = 115"
+        }
+        else {
+          Expand-Archive -Path "$packageName" -DestinationPath "$driverOutput" -Force 
+          Write-Host "ChromeDriver version < 115"
+        }           	
+      } 
+      DownloadChrome 
+    }
+    Start-Sleep -m 500
```
Comment (dloperab) [inline src/SEResourcesInstaller.ps1:151]: Add an enter

File: src/SEResourcesInstaller.ps1 [hunk 6]
```diff
@@ -34,173 +34,199 @@
...
+  # Evaluate the resource type to install
+  if ($componentName -eq "STD") {
+    [string]$myStandAloneURL = GetCompatibleResourceDownloadURL("STD")
+    $validationResult = IsValidURL($myStandAloneURL)
+
+    if ($validationResult -eq $true) {
+      DownloadPackageRaw $myStandAloneURL $componentName
+    }
+  }
+  if ($componentName -eq "CHR") {
```
Comment (dloperab) [inline src/SEResourcesInstaller.ps1:166]: Add enters between **if**

**Review by dloperab — CHANGES_REQUESTED** [review]
Please review the requested changes.

**Another question:** Have you performed tests locally?

**Comment by Carolina0802** [comment]
> Please review the requested changes.
> 
> **Another question:** Have you performed tests locally?

File: src/SEResourcesInstaller.ps1 (hunk 1 above)
Comment (Carolina0802) [reply src/SEResourcesInstaller.ps1]: corrected

File: src/SEResourcesInstaller.ps1 (hunk 6 above)
Comment (Carolina0802) [reply src/SEResourcesInstaller.ps1:166]: corrected

**Comment by Carolina0802** [comment]
> > Please review the requested changes.
> > **Another question:** Have you performed tests locally?

Yes, I have done local tests for both version cases.

**Comment by Carolina0802** [comment]
> Please review the requested changes.
> 
> **Another question:** Have you performed tests locally?

**Comment by Carolina0802** [comment]
> Please review the requested changes.
> 
> **Another question:** Have you performed tests locally?

**Comment by Carolina0802** [comment]
Yes, I have done local tests for both version cases

**Review by dloperab — APPROVED** [review]
LGTM @Carolina0802...Great work! 🚀

**Pull request merged**